Source/Plugins/NetworkEvents/NetworkEvents.cpp

```diff
@@ -263,14 +264,14 @@ String NetworkEvents::handleSpecialMessages(const String& s)
 
             if (key.compareIgnoreCase("Channel") == 0)
             {
-                //Make sure in range
+                // Make sure in range
                 if (value <= 8 && value >= 0)
```

Comment:
Just noticed this, but it should be `value <= 8 && value >= 1` if we're saying the valid range is 1-8.